homeassistant/components/dynalite/__init__.py

```diff
@@ -132,6 +133,7 @@ def validate_area(config: Dict[str, Any]) -> Dict[str, Any]:
             vol.Optional(CONF_NO_DEFAULT): cv.boolean,
             vol.Optional(CONF_CHANNEL): CHANNEL_SCHEMA,
             vol.Optional(CONF_PRESET): PRESET_SCHEMA,
+            vol.Optional(CONF_QUERY_CHANNEL): num_string,
```

Comment:
We no longer allow changes to the YAML configuration for integrations that communicate with devices or services.

Instead, this integration needs to be converted to a configuration flow first.

For more information see: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started with that: <https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Thanks 👍

Reply:
thanks. I started the process of configuring it via a config flow + the frontend stuff since it is quite a complex config for dynailte (unfortunately the Dynalite systems dont publish any configuration).
I have a PR with a config flow (https://github.com/home-assistant/core/pull/38828) and one that adds the websockets for the frontend to connect to (https://github.com/home-assistant/core/pull/38948), so waiting on these to convert to a config flow. There is also the first frontend PR (https://github.com/home-assistant/frontend/pull/6635) that is waiting on these to be merged. This means that in a short time, everything will be configurable via the UI
This specific PR is not urgent, but will need it at some point (perhaps after disabling the configuration.yaml option), as i run the entire config through the SCHEMA when it comes via the UI. In any case, this can wait until then if ok.
I assume the plan is that once an integration has a full config flow, we will disable the option to install it from yaml, right? If this is the plan, this PR can wait until after it is not accessible from the yaml.
thanks for reviewing.